chingu-voyage4/Geckos-Team-32
# Update to fix deployment styles bug

## Purpose
* Fix style bug on heroku

## Approach/Fixes
* Re-factor webpack config 
* Add babel-polyfill for other browsers
* Update index.html file for proper asset paths
* Refactor `public` directory to include a `dist` directory to house all static assets
  * This will not be pushed (included in .gitignore)

## Testing
* Download local PR
* Run `npm install` to add latest dependencies
* Run `npm run build:prod` for production build
* Check that app is working on dev-server
* Also check that app is working on local server
* Lastly check that app is working with mlabs
* Instructions for @odelavia 
  * Update the code and production build from this PR
  * Test everything on your local device first
  * Update heroku with new code
  * Check in heroku that proper styles are being used *after* logged in
* Instructions for @marylicious 
  * Update facebook/google callback URIs for heroku url when Oliver is ready
